test/apps/webgpu-line/app.tsx

```diff
@@ -0,0 +1,132 @@
+// deck.gl
```

Comment:
If these test apps are identical to the existing WebGL ones I think it's better to just add an option to the existing apps to run using webgpu (e.g. check for `#webgpu` in the URL, or add a toggle switch)

Reply:
> If these test apps are identical to the existing WebGL ones I think it's better to just add an option to the existing apps to run using webgpu (e.g. check for `#webgpu` in the URL, or add a toggle switch)

Yes, it will be very desirable to get to that point. But right now we would at least need to fix the async data loading bug in WebGPU and possible more issues. Again suitable for a fast follow in my opinion.